packages/pancake-uikit/src/components/Input/index.stories.tsx

```diff
@@ -39,3 +43,19 @@ export const Default: React.FC = () => {
     </div>
   );
 };
+
+export const Icons: React.FC = () => {
+  return (
+    <Box width="300px">
+      <InputGroup startIcon={<Search width="18px" />} endIcon={<StarFill width="18px" />} mb="24px" scale="sm">
```

Comment:
Wouldn't it better to make it like `<Input startIcon={...} endIcon={...} />`? Given that only possible child of InputGroup is Input it seems a bit unnecessary 🤔

Reply:
Yeah, perhaps I was over-thinking it too much. I didn't like the idea that `Input` with an icon will have a different root element than without (because you need a wrapper for the icons). Additionally I wanted to not have a wrapper for `Input`. However, I do agree that the implementation is clunky.

Let me revisit it and see if I can come up with something better.

Reply:
I think I will keep this format. Ideally we should do something like so 
```
<InputGroup>
  <InputStart>
    <SomeIcon />
  </InputStart>
  <Input />
  <InputEnd>
    <SomeIcon />
  </InputEnd>
</InputGroup>
```

Other libraries use this method but they also add `ids` to the component (internally) to identify what type of child it is.

I really want to avoid changing the root element.